### ISSUE TITLE
vertical metrics regressions

### ISSUE THREAD

**Issue by laerm0** [laerm0/opensans#15]
A request came in to add math glyphs to Open Sans so the whole Noto Sans math set has been added to Open Sans Roman (at the same weight – i.e. the glyphs look the same in all weights – these glyphs have not been added to italic). Because many of the math glyphs are large, the vertical metrics are now different.

Question: when the files are subset upon delivery, are the vert metrics recalculated? I assume not, but I was hoping if someone requests math Unicodes that that would be the only time we have giant YMax and YMin values.

Here are the extrema:

```
MinX g888 -1196.0
MinY summationWithIntegral -860.0
MaxX bottomcurlybracket 3471.0
MaxY bigReverseSolidus 2189.0
```

`g888` is a stacked accent, but the other three are math. 

Thought One for me is that maybe we could serve Open Sans Math as well? 

Opinions welcome from @davelab6 @m4rc1e @rsheeter and @garretrieger.

**Comment by m4rc1e** [comment]
Do you have a link for the original request?

**Comment by laerm0** [comment]
It was an email @davelab6 forwarded to me. 🤷‍♂

**Comment by m4rc1e** [comment]
Will be fixed in #16

**Comment by laerm0** [comment]
Closing as it is now fixed, thanks to @m4rc1e .
